code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc

```diff
@@ -0,0 +1,373 @@
+
+==== Types of Resource and Policy Blocks
+
+Each code category can generate either a resource or a policy block. For understanding the types of blocks corresponding to the code category see the table.
+
+[cols="1,1,1,1,1,1", options="header"]
+|===
+
+|Resource Type/ Code Category
+|IaC Misconfiguration
+|Vulnerabilities
+|Licenses
+|Secrets
+|Build Integrity
+
+|IaC Resource
+|✔️
+|✔️
+|✔️
+|
+|
+
+|Package
+|
+|✔️
+|✔️
+|
+|
+
+|File
+|
+|
+|
+|✔️
+|
+
+|Git Repository
+|
+|
+|
+|
+|✔️
+
+|Git Organization
+|
+|
+|
+|
+|✔️
+
+|CI/CD pipeline
+|
+|
+|
+|
+|✔️
+
+|===
+
+* *IaC Misconfiguration Resource Block*
++
+For each IaC misconfiguration issue, there is an extensive information in the resource block.
++
+image::proj-2.png[width=600]
++
+1. *Resource Name and Path*: Displays the resource name and it's code path.
++
+2. *Total number of Issues*: Displays the total number of issues identified in the resource.
++
+3. *Additional Information*: Displays columns of the information regarding the issue.
++
+* *Repository*: See the repository path.
++
+* *Policy*: See details on the non-conformant policy with the severity level.
++
+* *Labels*: Each issue has a corresponding label.
++
+** Has Fix: The issue will display this label if it has an automated fix provided by Prisma Cloud.
++
+** Custom Policy: The issue will display this label if it originated from a custom policy.
++
+* *Git User*: The name of last contributing Git user before identifying the issue.
++
+* *First Detected*: The timestamp of the issue when found.
+
+
+* *Vulnerabilities Resource Block*
++
+As a vulnerabilities issue, there is an extensive information in the resource block on vulnerable package.
++
+image::proj-3.png[width=800]
++
+1. *Package Name and Path*: Displays the package name and it's code path.
++
+2. *Total number of Issues*: Displays the total number of issues identified in the package.
++
+3. *Additional Information*: Displays columns of the information regarding the issue.
++
+* *CVE*: Displays the CVE name and the severity level of the violation.
++
+* *Package*: View the violated package while identifying if the package is a `Root` or a dependent `package`. If the CVE exposed by a dependent package, you can see the name of the dependent package.
++
+* *Root fix version*: Displays the recommended fix version for the root package to update.
++
+* *CVSS*: Displays the Common Vulnerability Scoring System (CVSS).
++
+* *Risk Factors*: Displays the risk factor of the CVE using Prisma Cloud defined values. The values range is Has Fix, Attach Complexity, DoS, Attack Vector, and Remote Execution.
++
+* *First Detected*: The timestamp of the issue when found.
+
+
+* *Secrets Resource Block*
++
+As a secrets issue, there is an extensive information in the resource block.
++
+image::proj-4.png[width=800]
++
+1. *Secret Name and Path*: Displays the repository name and it's code path.
```

Comment:
the "resource" here is a file so all secrets detected are in a file level and not repo